File: src/main/kotlin/a_basic_control_structures/exercise_14.kt
package a_basic_control_structures

/*
    The car rental company needs your help to charge for its services. Write a program that asks for the number of
    kilometers traveled by a rented car and the number of days it was rented for.
    Calculate the total price to be paid, knowing that the car costs R$90 per day and R$0.20 per kilometer driven.
 */

fun main() {
    print("km travelled: ")
    val kmTravelled = readln().toDouble()

    print("days rented: ")
    val daysRented = readln().toInt()

    val km = kmTravelled * 0.20
    val days = daysRented * 90

    val totalPrice = km + days
    val _totalPrice = String.format("%.2f", totalPrice)

    println("Km travelled: $kmTravelled; days rented: $daysRented; total price: $_totalPrice.")
}

File: Dart/a_basic_control_structure/exercise_14.dart
/*
    The car rental company needs your help to charge for its services. Write a program that asks for the number of
    kilometers traveled by a rented car and the number of days it was rented for.
    Calculate the total price to be paid, knowing that the car costs R$90 per day and R$0.20 per kilometer driven.
 */

import 'dart:io';

void main() {
  stdout.write('km travelled: ');
  double kmTravelled = double.parse(stdin.readLineSync()!);

  stdout.write('days rented: ');
  int daysRented = int.parse(stdin.readLineSync()!);

  double km = kmTravelled * 0.20;
  int days = daysRented * 90;

  double totalPrice = km + days;

  print(
      'km travelled: $kmTravelled, days rented: $daysRented, total price: ${totalPrice.toStringAsFixed(2)}');
}
